Automattic/woocommerce-payments
# Update `woocommerce-payments` package version number in `package-lock.json`

Fixes #4990 

#### Changes proposed in this Pull Request

Updates the `woocommerce-payments` package version number in `package-lock.json` to `4.9.0` to align with `package.json`.

#### Testing instructions

* Open `package.json`, and observe the version number is `4.9.0`.
* Open `package-lock.json`, and observe the version number is `4.9.0`.
* Run `npm install`, and observe no unstaged changes in git.

-------------------


**Post merge**
